download-center-upload/action.yml

```diff
@@ -25,6 +25,10 @@ inputs:
   artifact_file:
     description: Artifact(s) to upload. E.g. can contain a wildcard (*) or space separated string
     required: true
+  use_repo_name:
```

Comment:
I would suggest instead to have a parameter `repo_name` which by default falls back to the repo name (e.g. `camunda-optimize`) but can also be overwritten (with `optimize` for example). The if-else switch overwise is too verbose.

Reply:
I think if the repo name defaults to camunda-optimize we would still need an if else statement to check whether we are currently uploading Optimize files right? Because otherwise all other teams which use this action would upload in camunda-optimize so there would have to be some sort of logic in place to see whether we need to replace this default value

Reply:
yeah that makes sense actually 👍🏻